Mathlib/Order/IsNormal.lean

```diff
@@ -37,7 +37,7 @@ namespace IsNormal
 section LinearOrder
 variable [LinearOrder α] [LinearOrder β] [LinearOrder γ]
 
-/-- This condition is the LHS of the `IsLUB (f '' Iio a) (f a)` predicate. -/
+/-- This condition is the RHS of the `IsLUB (f '' Iio a) (f a)` predicate. -/
```

Comment:
Even after the change I find it hard to make sense of this doc string. Maybe we could just drop it?
```suggestion
```

Reply:
I do think the docstring is useful, it informs that this seemingly random condition is in fact a weakening of what the definition says.